src/components/Textarea/readme.md

```diff
@@ -118,6 +118,75 @@ const containerStyles = {
 />;
 ```
 
+##### Textarea with header
+
+```js
+import React, { useState } from 'react';
+import { Textarea, Chip } from 'react-rainbow-components';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faImage } from '@fortawesome/free-solid-svg-icons';
+import styled from 'styled-components';
+
+const StyledHeader = styled.div.attrs(props => {
+    return props.theme.rainbow.palette;
+})
+`
+    font-size: 12px;
+    color: ${props => props.text.header};
+    text-align: center;
+    border-radius: 0.875rem 0.875rem 0 0;
+    margin: 16px 16px 0 16px;
+`;
+
+const containerStyles = {
+    maxWidth: 700,
+};
+
+const Icon = styled.span.attrs(props => {
+    return props.theme.rainbow.palette;
+})`
+    ${props =>
+        `
+            color: ${props.brand.main};
+        `};
+`;
+
+function TextareaExample(props) {
+    const [count, setCount] = useState(0);
+    const maxLength = props.maxLength;
+    
+    return (
+        <Textarea
+            label="Textarea Label"
+            rows={4}
+            onChange={(event) => setCount(event.target.value.length)}
+            maxLength={maxLength}
+            placeholder="Type something . . ."
+            style={containerStyles}
+            header={
+                <StyledHeader>
+                    <Chip
+                        onDelete={() => alert('Delete Chip!')}
+                        style={{width:'100%'}}
```

Comment:
to be consistent with the example don't use styles inline, instead use a styled component